components/donation-form.tsx

```diff
@@ -31,6 +31,7 @@ import { useNetworkCheck } from '@/hooks/use-network';
 import { useDonationCallback } from '@/hooks/use-donation';
 import { useUsdcBalance } from '@/hooks/use-usdc-balance';
 import { PaymentSwitchWalletNetwork } from './payment/switch-wallet-network';
+import { useAuth } from '@/contexts/AuthContext';
```

Comment:
```suggestion
import { useAuth } from '@/contexts';
```
no need to specify the context file